test/test-integration/src/test/java/org/wso2am/micro/gw/tests/websocket/WebSocketTLSTestCase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
should be 2021. Add all other newly added files.

Reply:
fixed.